<launch>

  <rosparam command="load" file="$(find mpc_auto)/config/mpc_planning_params.yaml" />

  <node name="generate_waypoints" pkg="mpc_auto" type="generate_waypoints" respawn="true" output="screen" />
  <node name="waypoint_visualization" pkg="mpc_auto" type="waypoint_visualization" respawn="true" output="screen" />
  <node name="doMPC" pkg="mpc_auto" type="doMPC" respawn="true" output="screen" />

</launch>
